translator/trace/jaegerthrift_to_protospan_test.go

```diff
@@ -208,6 +208,20 @@ func TestConservativeConversions(t *testing.T) {
 						Code:    5,
 						Message: "cache miss",
 					},
+					Attributes: &tracepb.Span_Attributes{
+						AttributeMap: map[string]*tracepb.AttributeValue{
+							"http.status_code": {
+								Value: &tracepb.AttributeValue_IntValue{
+									IntValue: 5,
```

Comment:
In that case then let's use real status codes i.e. >=1XX

Reply:
Done.